button.html

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html PUBLIC "-//WC3//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/tr/xhtml1/DTD/xhtml1-transitional.dtd">
 <html>
-<head>
+<head> getElementById
```

Comment:
Delete `getElementById` altogether since you do not want to redefine it in your code.  Make sense?

Reply:
yes